File: data/sample/atcoder/abc104_a/C/impl_03.c
#include<stdio.h>
int main(){
	int rate;
	scanf("%d",&rate);
	
	if(rate<1200){
		printf("ABC");
	} else if(rate<2800){
		printf("ARC");
	}else{
		printf("AGC");
	}
	
	return 0;
}

File: data/sample/atcoder/abc104_a/Rust/impl_03.rs
#[allow(unused)]
use proconio::input;
#[allow(unused)]
use proconio::marker::Chars;
#[allow(non_snake_case)]
fn main() {
    input! { r:usize }
    match r {
        0..=1199 => println!("ABC"),
        1200..=2799 => println!("ARC"),
        _ => println!("AGC")
    }
}
